.storybook/main.js

```diff
@@ -5,6 +5,7 @@ const config = {
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
+    'storybook-addon-sass-postcss',
```

Comment:
Is this needed?

Reply:
taken from #5234 , thought it is required to load Sass files for stories